cucumber/features/devLoginPup.feature

```diff
@@ -0,0 +1,10 @@
+Feature: TTA Smarthub HSES login
+ 
+    Scenario: Welcome page is displayed
+        # Given https://tta-smarthub-dev.app.cloud.gov
+        Given the home page of tta-smarthub
+        Then we should see "Welcome to the TTA Smart Hub!" message
+    Scenario: Login is redirected to HSES
+        Given the home page of tta-smarthub
+        When pressing login
+        Then we should see "Head Start Enterprise System" page
```

Comment:
This is good, but ideally I'd like the scenarios to be a bit more like a first-person narrative. Something like:

```suggestion
        Given I'm on the Smart Hub home page
        When I press login
        Then I should see the HSES login page
```